packages/seacas/libraries/ioss/src/exodus/Ioex_ParallelDatabaseIO.C

```diff
@@ -2356,11 +2356,13 @@ int64_t ParallelDatabaseIO::get_field_internal(const Ioss::ElementSet *ns, const
 }
 
 int64_t ParallelDatabaseIO::get_field_internal(const Ioss::SideSet *fs, const Ioss::Field &field,
-                                               void * /* data */, size_t data_size) const
+                                               void *data, size_t data_size) const
 {
   size_t num_to_get = field.verify(data_size);
   if (field.get_name() == "ids") {
     // Do nothing, just handles an idiosyncrasy of the GroupingEntity
+    // However, make sure that the caller gets a consistent answer, i.e., don't leave the buffer full of junk
```

Comment:
I'll have to think about this...  buffer full of junk is faster; I need to look at when this is called and why...

Reply:
Addressed in branch `tkordenbrock-seacas/add-ioshell-compare`.